tests/data/config/fbs_config_good.py

```diff
@@ -0,0 +1,165 @@
+import numpy as np
+
+from lsst.sims.featureScheduler.modelObservatory import Model_observatory
+from lsst.sims.featureScheduler.schedulers import Core_scheduler
+from lsst.sims.featureScheduler.utils import standard_goals, Footprint
+import lsst.sims.featureScheduler.basis_functions as bf
+from lsst.sims.featureScheduler.surveys import Greedy_survey
+import lsst.sims.featureScheduler.detailers as detailers
+
+
+def gen_greedy_surveys(
+    nside=32,
+    nexp=1,
+    exptime=30.0,
+    filters=["r", "i", "z", "y"],
+    camera_rot_limits=[-80.0, 80.0],
+    shadow_minutes=60.0,
+    max_alt=76.0,
+    moon_distance=30.0,
+    ignore_obs="DD",
+    m5_weight=3.0,
+    footprint_weight=0.3,
+    slewtime_weight=3.0,
+    stayfilter_weight=3.0,
+    footprints=None,
+):
+    """
+    Make a quick set of greedy surveys
+
+    This is a convienence function to generate a list of survey objects that
+    can be used with lsst.sims.featureScheduler.schedulers.Core_scheduler.
+    To ensure we are robust against changes in the sims_featureScheduler
+    codebase, all kwargs are explicitly set.
+
+    Parameters
+    ----------
+    nside : int (32)
+        The HEALpix nside to use
+    nexp : int (1)
+        The number of exposures to use in a visit.
+    exptime : float (30.)
+        The exposure time to use per visit (seconds)
+    filters : list of str (['r', 'i', 'z', 'y'])
+        Which filters to generate surveys for.
+    camera_rot_limits : list of float ([-80., 80.])
+        The limits to impose when rotationally dithering the camera (degrees).
+    shadow_minutes : float (60.)
+        Used to mask regions around zenith (minutes)
+    max_alt : float (76.
+        The maximium altitude to use when masking zenith (degrees)
+    moon_distance : float (30.)
+        The mask radius to apply around the moon (degrees)
+    ignore_obs : str or list of str ('DD')
+        Ignore observations by surveys that include the given substring(s).
+    m5_weight : float (3.)
+        The weight for the 5-sigma depth difference basis function
+    footprint_weight : float (0.3)
+        The weight on the survey footprint basis function.
+    slewtime_weight : float (3.)
+        The weight on the slewtime basis function
```

Comment:
Nitpicking again but some of the doc strings end in a dot and others like this one don't.